<launch>
  <node pkg="ros_image_to_rtsp" type="proj_image_to_rtsp_cpp" name="proj_rtsp_sender" output="screen">
    <param name="config" value="$(find ros_image_to_rtsp)/config/config_proj.yaml"/>
    <param name="intrinsic" value="$(find ros_image_to_rtsp)/params/intrinsic1.csv"/>
    <param name="extrinsic" value="$(find ros_image_to_rtsp)/params/iou_optimized_transform1.txt"/>
    <param name="roi" value="$(find ros_image_to_rtsp)/roi/2025-04-22_17-19-59.pcd"/>
  </node>
</launch>
